lib/WebGLConstants.js

```diff
@@ -0,0 +1,300 @@
+"use strict";
```

Comment:
Talked offline, but you use Cesium's `defined`, `defaultValue`, and `WebGLConstants` instead for `modelMaterialsCommon`.